src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
     private Country country;
 
     public Actor() {
```

Comment:
According to the checklist, you should not create a default constructor when it is not needed. Since Hibernate uses reflection for creating objects, the default constructor is not required unless explicitly used in your code.